### ISSUE TITLE
material question

### ISSUE THREAD

**Issue by eroux** [buda-base/owl-schema#139]
I'm wondering what the drive is for the differentiation of MaterialLoktaPaper and MaterialDeshog ? It seems to be the same thing but with a different name...?

Also, dyed paper seems quite common and we have quite a few indication of indigo dye in the data of some partners, do you think we could model it?

**Comment by eroux** [comment]
Another interesting change in that direction: multi-layer paper is actually paper with poison (usually arsenic) applied on it too keep insects at distance. This would be another applied substance

**Comment by xristy** [comment]
When I was looking at the papers it appeared that Deshog from Bhutan and Lokta from Nepal are distinct even though both use the Daphne bush. The papermaking processes are apparently not the same so the resulting paper stock can be distinguished. I added reciprocal `rdfs:seeAlso` to the two. It would be interesting to ask Sam van Shaick about this.

I've added `bdr:AppliedMaterial_IndigoDye` and `bdr:MaterialMultiLayerPaper`
